pkg/cmd/triggertemplate/list_test.go

```diff
@@ -85,6 +85,12 @@ func TestListTriggerTemplate(t *testing.T) {
 			args:      []string{"list", "-n", "foo", "-o", "jsonpath={range .items[*]}{.metadata.name}{\"\\n\"}{end}"},
 			wantError: false,
 		},
+		{
+			name:      "TriggerTemplates from all namespaces",
```

Comment:
Please add a TriggerTemplate in a different namespace for this test. Currently, all the TriggerTemplates are in namespace `foo`.